### ISSUE TITLE
Newly added quiet option in say shell is a breaking change

### ISSUE THREAD

**Issue by xtreme-shane-lattanzio** [erikhuda/thor#751]
Hey! I am from the [LicenseFinder](https://github.com/pivotal/LicenseFinder) team that heavily relies on thor. I tried to track down why we are no longer getting output in newer versions of thor and found that [this](https://github.com/erikhuda/thor/commit/80fd49c0191b2618c9acb4639cb05666c684e586) commit is a breaking change. 

We used the quiet option to suppress basically everything except what is in say. With this, we are now getting empty string output. Was this intentional? Is there some other recommended way we should be outputting the results? We can try to change the way we do things on our end but wanted to ping here to figure out the reasoning for this change. Thanks!

**Comment by rafaelfranca** [comment]
Yes. It was deliberate and not considered a breaking change, since the fact that `say` was not respecting `quiet` was a bug. We could be adding a new option to force `say` to print. Do you want to open a PR?
